src/libcore/num/mod.rs

```diff
@@ -329,6 +329,27 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Returns the number of leading ones in the binary representation of `self`.
+
+# Examples
+
+Basic usage:
+
+```
+", $Feature, "let n = -1", stringify!($SelfT), ";
+
+assert_eq!(n.leading_ones(), 0);",
+$EndFeature, "
+```"),
+            #[unstable(feature = "leading_ones_trailing_ones", issue = "0")]
```

Comment:
Not on the libs team but it might be a good idea to open an issue so you have a non-zero number you can refer to here :)